### PR TITLE
Changes For App Publish

### PR DESCRIPTION
**Pull request by chiaramooney**
Remove Gesture Handler and Permissions Page. Gesture Handler rendered a blank page in Release mode. Permissions page creates need for privacy policy. 

Created visual assets for app. Modelled after this icon:
image

Updated app certificate to one with Microsoft as the publisher. 

Remove excess capabilities added by permissions page.

Upgraded CppWinrt version. Upgrade needed to pass WACK assessment.

### CODE REVIEW

File: windows/rngallery/Package.appxmanifest [hunk 1]
```diff
@@ -43,49 +43,12 @@
...
-    <DeviceCapability Name="gazeInput"/>
-    <DeviceCapability Name="location"/>
-    <DeviceCapability Name="lowLevel"/>
-    <DeviceCapability Name="microphone"/>
-    <DeviceCapability Name="proximity"/>
-    <DeviceCapability Name="pointOfService"/>
-    <DeviceCapability Name="webcam"/>
-  </Capabilities>
+ <Capabilities>
+  <Capability Name="internetClientServer"/></Capabilities>
```
Comment (chrisglein) [inline windows/rngallery/Package.appxmanifest:53]:
```suggestion
  <Capability Name="internetClientServer"/>
</Capabilities>
```

File: windows/rngallery/rngallery.vcxproj [hunk 2]
```diff
@@ -81,7 +81,17 @@
...
+  <PropertyGroup Label="UserMacros">
+    <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
+    <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
+    <AppxPackageSigningTimestampDigestAlgorithm>SHA256</AppxPackageSigningTimestampDigestAlgorithm>
+    <AppxAutoIncrementPackageRevision>False</AppxAutoIncrementPackageRevision>
+    <GenerateTestArtifacts>True</GenerateTestArtifacts>
+    <AppxBundle>Always</AppxBundle>
+    <AppxBundlePlatforms>x86|x64</AppxBundlePlatforms>
+    <HoursBetweenUpdateChecks>0</HoursBetweenUpdateChecks>
+    <PackageCertificateKeyFile>rngallery_Key.pfx</PackageCertificateKeyFile>
```
Comment (chrisglein) [inline windows/rngallery/rngallery.vcxproj:93]: This is back? Was this intended to be part of the PR?

File: windows/rngallery/rngallery.vcxproj.filters [hunk 3]
```diff
@@ -41,6 +38,141 @@
...
+    <Image Include="Assets\StoreLogo.scale-125.png">
+      <Filter>Assets</Filter>
+    </Image>
+    <Image Include="Assets\StoreLogo.scale-150.png">
+      <Filter>Assets</Filter>
+    </Image>
+    <Image Include="Assets\StoreLogo.scale-200.png">
+      <Filter>Assets</Filter>
+    </Image>
+    <Image Include="Assets\StoreLogo.scale-400.png">
```
Comment (chrisglein) [inline windows/rngallery/rngallery.vcxproj.filters:173]: I think you can do an include that's more like `Assets\*`. I... think.

File: src/RNGalleryList.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ import {DatePickerExamplePage} from './examples/DatePickerExamplePage';
 import {TimePickerExamplePage} from './examples/TimePickerExamplePage';
 //import {SketchExamplePage} from './examples/SketchExamplePage';
 import {SliderExamplePage} from './examples/SliderExamplePage';
-import {PermissionsExamplePage} from './examples/PermissionsExamplePage';
+//import {PermissionsExamplePage} from './examples/PermissionsExamplePage';
```
Comment (chrisglein) [inline src/RNGalleryList.ts:12]: For these disabled pages can you include a comment with the tracking issue number?
e.g.
```
// Disabled per issue #127
//import {PermissionsExamplePage} from './examples/PermissionsExamplePage';
```

File: windows/rngallery/rngallery.vcxproj (hunk 2 above)
Comment (chiaramooney) [reply windows/rngallery/rngallery.vcxproj:93]: I've never edited this part of the file, so it must be that VS is adding this info possibly when I create my app bundle. I'll remove it.

File: windows/rngallery/rngallery.vcxproj.filters (hunk 3 above)
Comment (chiaramooney) [reply windows/rngallery/rngallery.vcxproj.filters:173]: Ok I'll give that a try. This section was also automatically done by VS. I'll see if I can simplify the import.

Comment (chiaramooney) [reply windows/rngallery/rngallery.vcxproj.filters:173]: Just did a little digging and found this on the MS vcxproj docs "The Visual Studio C++ project system currently doesn't support wildcards or lists directly in project items. For example, these forms aren't supported: <None Include="*.txt"/>".

**Review by chrisglein — APPROVED** [review]

**Pull request merged**